src/tests/dbus-tests/test_loop.py

```diff
@@ -0,0 +1,99 @@
+#!/bin/python3
+
+import unittest
+import storagedtestcase
+import subprocess
+import time
+import dbus
+import glob
+import os
+
+class StoragedLoopDeviceTest(storagedtestcase.StoragedTestCase):
+    """This is SwapSpace related functions unit test"""
+
+    LOOP_DEVICE_FILENAME = 'loop_device.img'
+
+    def get_device(self, dev_name):
+        dev = self.get_object('', '/block_devices/' + os.path.basename(dev_name))
+        return dev
+    #enddef
+
+
+    def get_loop_iface(self, device):
```

Comment:
Is this really better than simply using `obj.Delete(self.no_options, dbus_interface=self.iface_prefix + '.Loop')`?